index.html

```diff
@@ -31,13 +31,13 @@
       <a href="#" class="logo">My logo</a>
       <ul>
         <li>
-          <a href="">Portfolio</a>
+          <a href="#portfolio">Portfolio</a>
         </li>
         <li>
-          <a href="">About</a>
+          <a href="#aboutme">About</a>
         </li>
         <li>
```

Comment:
You have done a great job, everything is as per the given Figma design. Nothing to be changed. It's time to merge.